R/read_delim.R

```diff
@@ -177,7 +177,7 @@ read_delimited <- function(file, tokenizer, col_names = TRUE, col_types = NULL,
     locale = locale)
 
   if (progress) {
-     print(spec, n = getOption("readr.num_columns", 20))
+     message(format(spec, n = getOption("readr.num_columns", 20)), appendLF = FALSE)
```

Comment:
Oh I wondered why I was getting the specs printed out now. I think it needs another line at top, e.g. "Parsed with column specification:"

And maybe if the number of specs is > n, print message about `spec()` function?